pandas/tests/io/pytables/test_store.py

```diff
@@ -4230,9 +4223,9 @@ def do_copy(f, new_f=None, keys=None, propindexes=True, **kwargs):
 
                         fd, new_f = tempfile.mkstemp()
 
-                    tstore = store.copy(
-                        new_f, keys=keys, propindexes=propindexes, **kwargs
-                    )
+                        tstore = store.copy(
+                            new_f, keys=keys, propindexes=propindexes, **kwargs
+                        )
```

Comment:
why the changed indentation?

Reply:
This was changed by running the 'black' command

Reply:
Haven't modified the existing structure here. What would you suggest to change it to?

Reply:
(in general, let the the person commenting/questioning hit the "resolve conversation" button)

if black did this, please undo it.

Reply:
Ah, noted re: the resolve conversation button, sorry about that. Will undo indentation now 👍